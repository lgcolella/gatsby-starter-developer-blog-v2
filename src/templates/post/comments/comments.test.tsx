/* Vendor imports */
import React from 'react'
import { shallow } from 'enzyme'
/* App imports */
import Comments from './index'

describe('Post > Comments component', () => {
  test('matches snapshot', () => {
    let wrapper = shallow(
      <Comments
        url='url/of/the/post'
        id='123'
        title='Title of the post'
      />
    )

    expect(wrapper).toMatchSnapshot()
  })
})
