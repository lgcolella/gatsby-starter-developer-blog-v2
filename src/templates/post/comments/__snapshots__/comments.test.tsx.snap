// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Post > Comments component matches snapshot 1`] = `
<Disqus
  config={
    Object {
      "identifier": "123",
      "title": "Title of the post",
      "url": "url/of/the/post",
    }
  }
/>
`;
